controllers/admin_main.php

```diff
@@ -16,7 +18,16 @@ public function preAction()
     {
         parent::preAction();
 
-        $this->structure->set('page_title', Language::_('AdminMain.index.page_title', true));
+        // Load required models
+        $this->uses(['Clients', 'Companies', 'ModuleManager', 'Packages', 'Invoices', 'Services']);
+        $this->helpers(['Form', 'CurrencyFormat']);
+        $this->components(['Session']);
+
+        // Set the client info
+        $client_id = ($this->get['client_id'] ?? ($this->get[0] ?? null));
+        $this->client = $this->Clients->get($client_id);
+
+        $this->structure->set('page_title', Language::_('AdminMain.index.page_title', true, $this->client->id_code));
```

Comment:
This will cause an error when calling AdminMain::getModuleFields().  You could set it only on non-ajax calls.